Header header


string command

int32 base_pwm
int32 shoulder_pwm
int32 elbow_pwm
int32 wrist_rot_pwm
int32 wrist_pwm
int32 gripper_pwm
